src/HaskellCI/GitHub/Yaml.hs

```diff
@@ -139,6 +139,12 @@ mapToYaml m = ykeyValuesFilt []
     | (k, v) <- M.toList m
     ]
 
+multiMapToYaml :: M.Map String [String] -> Yaml [String]
+multiMapToYaml m = ykeyValuesFilt []
```

Comment:
This is misleading. From the name one would expect this to generate a list, and not use `unlines`. Better to not use multiple paths at all

Reply:
Fine with me. I only did this so that I could pass multiple things to `path:` in the `cache` action, but if we're going to lump everything under `~/.haskell-ci-tools`, then I don't need to do this in the first place.